build.override.targets

```diff
@@ -3,37 +3,39 @@
 
   <PropertyGroup>
     <TraversalBuildDependsOn>
+      WcfCopyProjectJson;
       WcfSetup;
       $(TraversalBuildDependsOn);
       WcfCleanup;
     </TraversalBuildDependsOn>
   </PropertyGroup>
-  
+
   <PropertyGroup>
     <TraversalCleanDependsOn>
       WcfClean;
       $(TraversalCleanDependsOn);
     </TraversalCleanDependsOn>
   </PropertyGroup>
-  
+
   <PropertyGroup>
     <WcfSelfHostSetupLog>$(MSBuildThisFileDirectory)\SelfHostedWcfServiceSetup.log</WcfSelfHostSetupLog>
     <WcfSelfHostCleanupLog>$(MSBuildThisFileDirectory)\SelfHostedWcfServiceCleanup.log</WcfSelfHostCleanupLog>
+    <ScenarioLatestDependencyJson>$(MSBuildThisFileDirectory)\src\System.Private.ServiceModel\tools\test\JsonFiles\stable.dependencies.json</ScenarioLatestDependencyJson>
+    <UseStableDependencies Condition="'$(UseStableDependencies)'==''">true</UseStableDependencies>
```

Comment:
I agree we should rename to $(UseLatestDependencies) and default to false.  It makes it clearer that you opt-in to latest dependencies (whereas the current name suggests you are only opting out of stable and not clear what that implies).  It would also set a precedent if we ever introduce something like $(UseLastKnownGoodDependencies) in the future.